src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -871,12 +875,19 @@ internal SignatureHelp GetSignatureHelp(TextDocumentPosition textDocumentPositio
                                    startLine + 1,
                                    endColumn + 1,
                                    bindingContext.MetadataDisplayInfoProvider);
-                                
-                                // convert from the parser format to the VS Code wire format
-                                return AutoCompleteHelper.ConvertMethodHelpTextListToSignatureHelp(methods, 
-                                    methodLocations,
-                                    startLine + 1,
-                                    endColumn + 1);
+
+                                if (methodLocations != null)
```

Comment:
>if (methodLocations != null) [](start = 32, length = 28)

you may push the null check into the helper method..this method is already pretty long so additional conditionals and returns adds to that complexity a little.